packages/clarity-js/src/layout/dom.ts

```diff
@@ -367,11 +364,11 @@ function getFullUrl(relative: string): string {
     return a.href;
 }
 
-function getNextId(node: Node): number {
+function getPreviousId(node: Node): number {
     let id = null;
-    while (id === null && node.nextSibling) {
-        id = getId(node.nextSibling);
-        node = node.nextSibling;
+    while (id === null && node.previousSibling) {
```

Comment:
What happens when there are nodes witthout ids but are siblings? This skips them. Can you add a comment explaining why that is OK?

Reply:
Yes adding a comment. But these nodes are the ones that we skip while walking the DOM like comment tags and script tags.